client/app/scripts/superdesk-archive/module.js

```diff
@@ -275,10 +275,11 @@ define([
                                 });
                         }]
                 })
-                .activity('Re-write', {
+                .activity('rewrite', {
                     label: gettext('Re-write'),
                     icon: 'multi-star-color',
                     filters: [{action: 'list', type: 'archive'}],
+                    group: 'corrections',
```

Comment:
Is the `group: 'corrections'` for grouping post publish action such as correction, kill, re-write and re-open (which is new take on the published item)?  If yes, then we would need `group: 'corrections'` for other actions as well.